### DISCUSSION TITLE
Documentation website off

### DISCUSSION THREAD

**Discussion by lqmorais** [apexcharts/apexcharts.js#3386] [General]
Since yesterday the documentation website is unavailable 😢

Is this problem occurring for everyone?

**Comment by brianlagunas** [comment]
If you ping the apex website what IP address do you get?

**Reply by lqmorais** [reply]
I get this:  `2606:4700:3031::6815:3817`

**Reply by jaden** [reply]
I was having the same issue and found it's only when using IPv6. Going directly to the IPv4 site works (apart from the SSL error) https://100.24.77.146/

An [SSL check](https://www.ssllabs.com/ssltest/analyze.html?d=apexcharts.com) also got "Failed to communicate with the secure server" for the two IPv6 addresses, but was able to test the cert fine over IPv4.

**Reply by brianlagunas** [reply]
That was the info I needed! I think I got it figured out. Everything should be working now.

**Reply by jaden** [reply]
Thanks, that fixed it!

**Reply by lqmorais** [reply]
Thanks, that fixed for me too! 😊
